Update set-up-your-team-hierarchy.md (#9457)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/set-up-your-team-hierarchy.md

## Apply your hierarchy

> [!NOTE]

> > To perform this step, you'll use "Microsoft Teams" PowerShell cmdlets. You can use version 4.6.0 or later of the Microsoft Teams cmdlets. This also applies for Government Community Cloud (GCC) customers.

Reviewer: ```suggestion
> To perform this step, you must use Microsoft Teams PowerShell cmdlets. You should use version 4.6.0 or later of the Microsoft Teams cmdlets. This also applies to Government Community Cloud (GCC) customers.
```
